packages/react-core/src/components/Breadcrumb/BreadcrumbItem.tsx

```diff
@@ -18,6 +26,8 @@ export interface BreadcrumbItemProps extends React.HTMLProps<HTMLLIElement> {
   target?: string;
   /** Sets the base component to render. Defaults to <a> */
   component?: React.ElementType;
+  /** A render function to render the component inside the breadcrumb item. */
+  render?: (props: BreadcrumbItemRenderProps) => JSX.Element;
```

Comment:
This is inconsistent with how we defined render prop in the Label component.
@jshuler any thoughts on this vs the other implementation?

Reply:
No problem, I can change the return value to `React.ReactNode`. 
If there are other props that are missing or redundant I can also update as well.

Reply:
Looking at this SO it seems ReactNode would be more accurate for a render function

https://stackoverflow.com/questions/58123398/when-to-use-jsx-element-vs-reactnode-vs-reactelement